From fd7627442b3581e97d1275ae7e117d6521fe6500 Mon Sep 17 00:00:00 2001
From: "Tom G. Christensen" <tgc@jupiterrise.com>
Date: Mon, 6 Jun 2022 10:56:04 +0200
Subject: [PATCH 06/12] No pthread support for Solaris 2.6

Solaris 2.6 only conforms to POSIX.1c pthreads and does not support the
PTHREAD_MUTEX_RECURSIVE attribute.
---
 config.mak.uname | 1 +
 1 file changed, 1 insertion(+)

diff --git a/config.mak.uname b/config.mak.uname
index 29af78fa33..c458f7425f 100644
--- a/config.mak.uname
+++ b/config.mak.uname
@@ -194,6 +194,7 @@ ifeq ($(uname_S),SunOS)
 		NO_SETENV = YesPlease
 		NO_STRLCPY = YesPlease
 		NO_STRTOUMAX = YesPlease
+		NO_PTHREADS = YesPlease
 		GIT_TEST_CMP = cmp
 	endif
 	ifeq ($(uname_R),5.7)
-- 
2.36.1

